tests/kafkatest/services/kafka/kafka.py

```diff
@@ -242,8 +243,8 @@ def start_node(self, node):
     def pids(self, node):
         """Return process ids associated with running processes on the given node."""
         try:
-            cmd = "ps ax | grep -i kafka | grep java | grep -v grep | awk '{print $1}'"
-
+            cmd = "jcmd | grep -e org.apache.zookeeper.server.quorum.QuorumPeerMain -e kafka.Kafka" \
```

Comment:
I wanted to maintain the behavior of the existing code where it kills both Kafka and ZK, in case it was somehow required.  But thinking about it a little bit more, I will try changing the Kafka Service to just clean up Kafka, and see if that causes any failures.  In theory the ZK service should clean up the ZK instance it starts.